client/app/account/billing/autoRenew/warnNicBilling/billing-autoRenew-warnNicBilling.html

```diff
@@ -0,0 +1,10 @@
+<div data-ng-controller="billingControllersAutoRenewWarnNicBillingCtrl as $ctrl">
```

Comment:
Component ? :smile:

Reply:
I'm not sure I can switch to a component as it still uses the legacy modal system